internal/security/proxy/service_test.go

```diff
@@ -131,10 +132,75 @@ func TestInit(t *testing.T) {
 
 	mockCertPair := bootstrapConfig.CertKeyPair{Cert: "test-certificate", Key: "test-private-key"}
 	mockLogger := logger.MockLogger{}
-	service := NewService(NewRequestor(true, 10, "", mockLogger), mockLogger, configuration)
-	err = service.Init(mockCertPair)
-	if err != nil {
-		t.Errorf(err.Error())
+
+	origProxyRoutEnv := os.Getenv(AddProxyRoutesEnv)
+	defer func() {
+		_ = os.Setenv(AddProxyRoutesEnv, origProxyRoutEnv)
+	}()
+
+	tests := []struct {
+		name               string
+		proxyRouteEnvValue string
+		expectNumRoutes    int
+		expectError        bool
```

Comment:
This is never `true` in your test case. Need to added some error test cases for parsing bad additional

Reply:
yea, but I probably will remove it...since if there is error on the additional env. we probably do not want to fail everything (at least not failing for the ones from configuration file). so this will be always false in terms of additional env adding.

Reply:
I have logged the error in the parsing error cases.

Reply:
Yep, agreed.